org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -96,22 +96,24 @@ class Translator {
                                                     }
                                                     break;
                                             }
-                                        } else {
-                                            console.log(deviceStatus);  //failed to get device status
                                         }
                                         
                                         // Create a translator for this device and get the platform information, possibly expanded
                                         return OpenT2T.createTranslatorAsync(opent2tInfo.translator, { 'deviceInfo': deviceInfo, 'hub': this })
                                             .then((translator) => {
-
                                                 // Use get to translate the Insteon formatted device that we already got in the previous request.
                                                 // We already have this data, so no need to make an unnecesary request over the wire.
                                                 return OpenT2T.invokeMethodAsync(translator, opent2tInfo.schema, 'get', [expand, deviceData])
                                                     .then((platformResponse) => {
                                                         return platformResponse;
                                                     });
+                                            }).catch((err) => {
+                                                console.log('warning: OpenT2T.createTranslatorAsync error - ' + err);
+                                                return Promise.resolve(undefined);
                                             });
```

Comment:
Oh, I see that it's a log with a promise.resolve, not a reject.  In that case leave it as is.

---
In reply to: [103301882](https://github.com/openT2T/translators/pull/187#discussion_r103301882) [](ancestors = 103301882)